### DISCUSSION TITLE
Change kickoff icon

### DISCUSSION THREAD

**Discussion by lgoette** [nix-community/plasma-manager#148] [Q&A] [answered]
Is there a way to change the kickoff icon?
The icon is set in the "plasma-org.kde.plasma.desktop-appletsrc" file but the value is in a random place:
```
[Containments][2][Applets][3][Configuration][General]
favoritesPortedToKAstats=true
icon=nix-snowflake
systemFavorites=suspend\\,hibernate\\,reboot\\,shutdown
```
Is there a way to define values in a spec with multiple values in square brackets or a simpler way?
I also dont know if the numbers in the square brackets are predictable :/

**Comment by magnouvean** [comment, marked as answer]
Everything to do with panels in teh `plasma-org.kde.plasma.desktop-appletsrc` is configured with the panels module. The following should work as a widget for the panel:
```nix
      {
        name = "org.kde.plasma.kickoff";
        config = {
          General.icon = "nix-snowflake-white";
        };
      }
```
Where you replace `nix-snowflake-white` with your desired icon (once the panels module becomes functional again).

**Reply by lgoette** [reply]
That worked great. 
I just stumbled on a special case when I was configuring more: When I wanted to change the configuration of the System Tray, I didn't work like that. When I lokked at the `plasma-org.kde.plasma.desktop-appletsrc` file, I saw that some settings like `iconSpacing` are configured in `org.kde.plasma.private.systemtray`. But the widget name when adding the Tray is `org.kde.plasma.systemtray`. Is there a way to configure that?

**Reply by magnouvean** [reply]
Configuring the systray isn't really well supported yet sadly due to it being separate from the panels. Support for it will eventually be implemented though I reckon.
